### PR TITLE
Jetpack: 11.2-a.5 changelog editorial

### PR DESCRIPTION
**Pull request by coder-karen**
#### Changes proposed in this Pull Request:

* Jetpack 11.2-a.5 changelog and readme edited and ready for proof-reading.

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Proof-read.

### CODE REVIEW

File: projects/plugins/jetpack/CHANGELOG.md [hunk 1]
```diff
@@ -4,35 +4,24 @@
...
-- Enable Jetpack block collection for the native editor block inserter. [#25092]
-- Jetpack: handle Loading state listeging vp events [#25096]
-- Jetpack: iterate over VPBlock UI errors [#25064]
-- Jetpack: replace fallback thumbail by a Loading element [#25088]
-- Jetpack: show Mediaplceholder errors in the VPBlock [#25069]
-- Jetpack: use BlockIcon in Placeholder cmp - vpblock [#25062]
-- Jetpack: VPBlock. Clean obsolte attr. Janitorial. [#25095]
+- Blocks: enable Jetpack block collection for the native editor block inserter (on self hosted Jetpack sites). [#25092]
+- Connection: make sure pre-existing settings are respected on plugin activation. [#24980]
+- VideoPress: add UX improvements including fallback thumbnail replaced by loading element, and better error messaging (in Beta only). [#25088, #25064]
```
Comment (bindlegirl) [inline projects/plugins/jetpack/CHANGELOG.md:9]: Should VP beta stuff go to Other changes?

File: projects/plugins/jetpack/readme.txt [hunk 2]
```diff
@@ -242,16 +242,19 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
...
 
 == Changelog ==
-### 11.2-a.3 - 2022-07-12
+### 11.2-a.5 - 2022-07-19
 #### Enhancements
-- Podcast Player: add new actions to make it possible for users to set up code that runs for podcast fetches.
-- VideoPress Block: add enhancements such as an upload progress bar, improved UX, and limited preview attempts (available in Beta only).
+- Blocks: enable Jetpack block collection for the native editor block inserter (on self hosted Jetpack sites).
+- Connection: make sure pre-existing settings are respected on plugin activation.
+- VideoPress: add UX improvements including fallback thumbnail replaced by loading element, and better error messaging (in Beta only). [#25088, #25064]
```
Comment (bindlegirl) [inline projects/plugins/jetpack/readme.txt]:
```suggestion
- VideoPress: add UX improvements including fallback thumbnail replaced by loading element, and better error messaging (in Beta only).
```

File: projects/plugins/jetpack/CHANGELOG.md (hunk 1 above)
Comment (coder-karen) [reply projects/plugins/jetpack/CHANGELOG.md:9]: We left it in enhancements in [11.2-a.3](https://github.com/Automattic/jetpack/pull/25073/files) so I think fine (can still be tested with Beta afaik)

File: projects/plugins/jetpack/readme.txt (hunk 2 above)
Comment (coder-karen) [reply projects/plugins/jetpack/readme.txt]: I've always wondered about that but just copied what I saw previously ([like here](https://github.com/Automattic/jetpack/pull/24674/files)). Better to remove do you think?

Comment (bindlegirl) [reply projects/plugins/jetpack/readme.txt]: We remove PR numbers from the readme file, right?

Comment (coder-karen) [reply projects/plugins/jetpack/readme.txt]: Yeah.

Comment (coder-karen) [reply projects/plugins/jetpack/readme.txt]: Oh! This is readme. Sorry, haha. Will fix that.

**Review by bindlegirl — APPROVED** [review]

**Comment by coder-karen** [comment]
r249254-wpcom

**Pull request merged**